src/nvim/getchar.c

```diff
@@ -3435,7 +3435,8 @@ static void showmap(mapblock_T *mp, bool local)
   size_t len = 1;
 
   if (message_filtered(mp->m_keys)
-      && mp->m_str != NULL && message_filtered(mp->m_str)) {
+      && (mp->m_str[0] == NUL || message_filtered(mp->m_str))
```

Comment:
Why add this `mp->m_str[0] == NUL` check?

Reply:
Not required I'll remove it

Reply:
Removed.